### PR TITLE
add documentation for using CLI to locally deploy and test teleporter ready subnets

### PR DESCRIPTION
**Pull request by felipemadero**
# Docs PR Template

## Why this should be merged
A new functionality is added to CLI to support deploying subnets altogether with teleporter contracts
and relayer process.

## How this works
A how to is added that documents this for local networks.

## How these changes were tested 
They were copying from direct command execution

## To prevent any errors while building

- [] run `vale /docs/file/path` on all changed `.md` files to ensure all grammar rules pass
- [] run `markdownlint /docs/file/path` on all changed `.md` files to ensure all linting rules pass
- [] complete the above two checks and all additional rules outlined in `style-checker-notes.md` to
  ensure all checks pass

### If a document was removed/deleted

- [] the path to that doc must be redirected to a valid URL via the
  `_redirects` file
- [] `_redirects` were manually verified with the cloudflare preview link
- [] `sidebars.json` reflects all changes made to file path

### If a document was moved

- [] all files that were moved from their current directory to a new path have had their paths
  redirected via the `_redirects` file
- [] `_redirects` were manually verified with the cloudflare preview link
- [] `sidebars.json` reflects all changes made to file path

### CODE REVIEW

File: docs/tooling/cli-cross-chain/teleporter-on-local-networks.md [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+- the Teleporter contracts are also deployed to local `C-Chain`
+- a Teleporter relayer is executed in background
+
+As part of the deploy, an [AWM Relayer](https://github.com/ava-labs/awm-relayer) is installed, configured and executed.
+A relayer, or some other intermediate procedure, is always needed for a Teleporter or warp message to be sent from
+one Subnet to the other. In this case the relayer is an actor that listens for new messages being generated on a source Subnet,
+and send them to the destination Subnet.
+
+On local network, CLI configures the relayer so every Subnet can message to every other Subnet.
+
```
[the remark this answers is not in the document]
Comment (felipemadero) [reply docs/tooling/cli-cross-chain/teleporter-on-local-networks.md:128]: if the user add a 3rd subnet, relayer will be reconfigured, relaunched

**Comment by meaghanfitzgerald** [comment]
/gt docs/build/subnet/deploy/fuji-testnet-subnet.md i18n/es/docusaurus-plugin-content-docs/current/build/subnet/deploy/fuji-testnet-subnet.md spanish

/gt docs/build/subnet/deploy/mainnet-subnet.md i18n/es/docusaurus-plugin-content-docs/current/build/subnet/deploy/mainnet-subnet.md spanish

/gt docs/build/subnet/upgrade/considerations-subnet-upgrade.md i18n/es/docusaurus-plugin-content-docs/current/build/subnet/upgrade/considerations-subnet-upgrade.md spanish

File: i18n/es/docusaurus-plugin-content-docs/current/build/subnet/deploy/fuji-testnet-subnet.md [hunk 2]
```diff
@@ -85,7 +85,7 @@ La respuesta debería verse algo como:
 ```
 
 Esa parte que dice, `NodeID-5mb46qkSBj81k9g9e4VFjGGSbaaSLFRzD` es el NodeID, todo el conjunto.
-El usuario va a necesitar este ID en la sección posterior al llamar a [addValidator](#agregar-un-validador).
+El usuario va a necesitar este ID en la sección posterior al llamar a [addValidator](#add-a-validator).
```
Comment (meaghanfitzgerald) [inline i18n/es/docusaurus-plugin-content-docs/current/build/subnet/deploy/fuji-testnet-subnet.md:88]:
```suggestion
El usuario va a necesitar este ID en la sección posterior al llamar a addValidator.
```

File: docs/build/subnet/deploy/mainnet-subnet.md [hunk 3]
```diff
@@ -173,8 +173,8 @@ After that, CLI shows the `Mainnet` Ledger address used to fund the deployment:
 Ledger address: P-avax1ucykh6ls8thqpuwhg3vp8vvu6spg5e8tp8a25j
 ```
 
-The deployment requires running a [createSubnet transaction](/reference/avalanchego/p-chain/api.md#platformcreatesubnet)
-and a [createBlockchain transaction](/reference/avalanchego/p-chain/api.md#platformcreateblockchain),
+The deployment requires running a `createSubnet` transaction
+and a `createBlockchai` transaction,
```
Comment (meaghanfitzgerald) [inline docs/build/subnet/deploy/mainnet-subnet.md:177]:
```suggestion
and a `createBlockchain` transaction,
```

File: docs/tooling/cli-cross-chain/teleporter-on-local-networks.md [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+machines support Teleporter.
+
+## Prerequisites
+
+- [Avalanche-CLI installed](/tooling/cli-guides/install-avalanche-cli.md)
+
+## Create Subnets Configurations
+
+Let's create a Subnet called `<subnet1Name>` with the latest Subnet-EVM version, a chain ID of 1, TOKEN1 as the token name,
+and with default Subnet-EVM parameters (more information regarding Subnet creation can be found [here](/build/subnet/hello-subnet#create-your-subnet-configuration)):
```
Comment (meaghanfitzgerald) [inline docs/tooling/cli-cross-chain/teleporter-on-local-networks.md:31]:
```suggestion
and with default Subnet-EVM parameters (more information regarding Subnet creation can be found [here](/build/subnet/hello-subnet.md#create-your-subnet-configuration)):
```

**Comment by meaghanfitzgerald** [comment]
/gt docs/build/subnet/deploy/mainnet-subnet.md i18n/es/docusaurus-plugin-content-docs/current/build/subnet/deploy/mainnet-subnet.md spanish

**Comment by meaghanfitzgerald** [comment]
/gt docs/build/subnet/upgrade/considerations-subnet-upgrade.md i18n/es/docusaurus-plugin-content-docs/current/build/subnet/upgrade/considerations-subnet-upgrade.md spanish

File: i18n/es/docusaurus-plugin-content-docs/current/build/subnet/deploy/mainnet-subnet.md [hunk 5]
```diff
@@ -335,168 +319,168 @@ Al presionar `Sí`, se escribe el archivo necesario:
...
 ```text
-Sus claves de autenticación de Subnet para la creación de la TX de add validator: [P-avax1ucykh6ls8thqpuwhg3vp8vvu6spg5e8tp8a25j]
+Tus claves de autenticación de la Subnet para la creación de la transacción de agregar validador: [P-avax1ucykh6ls8thqpuwhg3vp8vvu6spg5e8tp8a25j]
 ```
 
-### Establecer NodeID
+### Establece el NodeID
 
-Ahora ingrese el [**NodeID**](#obtener-tus-nodeids-de-mainnet) del validador.
+Ahora ingresa el [**NodeID**](#obteniendo-tus-nodeids-de-mainnet) del validador.
```
Comment (meaghanfitzgerald) [inline i18n/es/docusaurus-plugin-content-docs/current/build/subnet/deploy/mainnet-subnet.md:405]:
```suggestion
Ahora ingrese el [**NodeID**](#obtener-tus-nodeids-de-mainnet) del validador.
```

**Review by meaghanfitzgerald — APPROVED** [review]
lgtm. there may be some tutorials in the cli tab that contain info about [deprecated endpoints](https://github.com/ava-labs/avalanchego/releases/tag/v1.11.2). Would appreciate a tutorial refresh from the CLI team.

**Pull request merged**